android/assets/jsons/Civ V - Vanilla/Policies.json

```diff
@@ -403,34 +403,35 @@
 			},
 			{
 				"name": "Militarism",
-				"uniques": ["Gold cost of purchasing units -33%"],
+				"uniques": ["Gold cost of purchasing [All] units -[33]%"],
 				"row": 1,
 				"column": 5
 			},
 			{
 				"name": "Fascism",
-				"uniques": ["Quantity of strategic resources produced by the empire increased by 100%"],
+				"uniques": ["Quantity of strategic resources produced by the empire +[100]%", "+[2] Movement for all [Great General] units"],
 				"requires": ["Populism","Militarism"],
 				"row": 2,
 				"column": 3
 			},
 			{
 				"name": "Police State",
 				"uniques": ["[+3 Happiness] from every [Courthouse]", "+[100]% Production when constructing [Courthouse]"],
+				// There are also some uniques regarding espoinage, which as of this writing is not yet implemented
 				"requires": ["Militarism"],
 				"row": 2,
 				"column": 5
 			},
 			{
 				"name": "Total War",
-				"uniques": ["+[15]% Production when constructing [Military] units [in all cities]", "New [Military] units start with [15] Experience"],
+				"uniques": ["+[25]% Production when constructing [Military] units [in all cities]", "New [Military] units start with [15] Experience"],
 				"requires": ["Police State","Fascism"],
 				"row": 3,
 				"column": 4
 			},
 			{
 				"name": "Autocracy Complete",
-				"uniques": ["+20% attack bonus to all Military Units for 30 turns"]
+				"uniques": ["+[25]% attack bonus to all Military Units for [50] turns"]
```

Comment:
Unitfilter here, while you're at it ;)
I hope this isn't too annoying for you, but if we already need to change uniques in both mods and translations, let's not have to do it again in the future...

Reply:
No worries, I'll update it this evening